static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -187,70 +169,87 @@ define([
     // Rather than calling this directly, use spaceFiller to inject content into the page.
     // SpaceFiller will safely queue up all the various asynchronous DOM actions to avoid any race conditions.
     function findSpace(rules) {
-        var body;
+        var body, getDimensions;
 
-        rules = rules || defaultRules;
+        rules || (rules = defaultRules);
         body = rules.bodySelector ? document.querySelector(rules.bodySelector) : document;
+        getDimensions = rules.absoluteMinAbove ? _mapElementToComputedDimensions : _mapElementToDimensions;
```

Comment:
To only load ads below the viewport, we'll be using offsets relative to the viewport rather than to the parent container.